### ISSUE TITLE
Request - finding catalog records without media

### ISSUE THREAD

**Issue by wellerjes** [ArctosDB/arctos#9418]
Help us understand your request (check below):

- [ ] search or data request (help with SQL or provide the specific fields you are trying to retrieve for your collection)
- [ ] add a new Github user to the Arctos Users team
- [ ] a new barcode series
- [ ] a new term for data entry or management 
- [ ] a change or new report or label
- [ ] update to an Agent (split, merge, or other type of change)
- [ ] bulkload changes (to Agents, Identifiers, etc that are not already available as a bulkloading tool)
- [x] new feature or otherwise a good idea to make Arctos even better
- [ ] quarantine taxon name and suggested replacement (please supply links)
- [ ] API key request (see https://handbook.arctosdb.org/documentation/api.html for requirements)
- [ ] error or bug (please include enough specific information to reproduce)
- [ ] general inquiry (or email arctoscontact@googlegroups.com)
- [ ] access issue (see https://arctosdb.org/faq/)
- [ ] other

**Describe what you're trying to do** 
Is there a way to easily pull records that do not have media attached? We are trying to determine how many specimens we need to image, but my current workaround for determining this is to do a general search, download the data, then search of records that do have images, download that data, then add the GUIDs from both downloads to an Excel sheet and delete everything that has a duplicate. Is there an easier way? If not, I'd like to add it to future user experience discussions.

Thanks!

**Comment by dustymc** [comment]
>do not have media attached

Please be specific. Do you mean ``% cataloged_item`` attached, less directly, any particular relationship(s)??

>search of records that do have images

This suggests only https://arctos.database.museum/info/ctDocumentation.cfm?table=ctmedia_relationship#shows_cataloged_item ??

**Comment by wellerjes** [comment]
> Please be specific.

I would like an option like "NULL" for media; there is absolutely no media record attached, meaning that it needs to be imaged.

> search of records that do have images

Yes, my workaround only brings up media that has "shows_cataloged_item" associated with it.

**Comment by Jegelewicz** [comment]
@wellerjes Dusty wants to know if you consider media attached via an accession or some other method than "shows cataloged item" should be considered "has media attached".

**Comment by wellerjes** [comment]
> media attached via an accession

We're looking to determine if the specimen/object does not have "Catalog Record Media" / "Media linked to the Cataloged Item"; the search should disregard if the record has media from Transactions / Accessions.

Thanks!
